Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -34,6 +34,11 @@
     subjectDirectory = fullfile(imPar.BidsRoot,['sub-' bidsLabel.subject]);
     if ~xASL_exist(subjectDirectory,'dir')
```

Comment:
You can simplify this by moving `xASL_adm_CreateDir` outside, and below, the if statement.
`xASL_adm_CreateDir` always checks if a folder exists before creating it.

Reply:
10deab0